docker-compose.yml

```diff
@@ -26,6 +26,8 @@ services:
     depends_on:
       "postgres":
         condition: service_healthy
+      "ipfs":
+        condition: service_healthy
```

Comment:
I can't find you added `healthcheck` for ipfs, unless the image has built in it, this might not work.

Reply:
Yes, the ipfs image has the built in `healthcheck` command.